scripts/touchpoint_api.py

```diff
@@ -0,0 +1,106 @@
+from argparse import ArgumentParser
+import requests
+import json
+import re
+
+# This script calls the touchpoint API to retrieve survey results for the provided document (form_id)
+# You will need to use your own Touchpoint API key:
+# touchpoint.py <form_id> <api_key> <start_date> <end_date>
+# if you want to save to the default file (touchpoint.csv), use the -p flag.
+
+
+
+MAX_PAGES = 500 # this can be changed to throttle if needed
+CSV_HEADER = 'id, date, referer, user agent, page, raw answer, language, english answer\n'
+
+def parse_args():
+    parser = ArgumentParser(
+        fromfile_prefix_chars="@", prog='Touchpoint Parser',
+        description='Parse specific results from a Touchpoint form',
+        epilog='Now, be off and about your business!', conflict_handler='error', add_help=True
+    )
+    parser.add_argument("form_id")
+    parser.add_argument("apikey")
+    parser.add_argument("start_date")
+    parser.add_argument("end_date")
+    parser.add_argument("filename")
+    parser.add_argument("-s", "--show", action="store_true") # to print results to console
+    parser.add_argument("-p", "--persist", action="store_true") # to save the results to the filename provided
+    return parser.parse_args()
+
+def make_single_call(base_url, page_number, start_date, end_date, api_key):
+    try:
+        response = requests.get(
+            f'{base_url}{page_number}&start_date={start_date}&end_date={end_date}',
+            headers={'X-Api-Key': api_key}
+        )
+        response.raise_for_status()
+        return response.json()
+    except requests.exceptions.RequestException as e:
+        print(f"Error making API request: {e}")
+        return None
+
+def strip_language(source):
+    if '/es/' in source:
+        return 'es'
+    elif '/fr/' in source:
+        return 'fr'
+    else:
+        return 'en'
+
+def translate(response):
+    translation_dict = {
+        'Si': 'yes', 'si': 'yes', 'yes': 'yes', 'Yes': 'yes', 'Oui': 'yes', 'no': 'no', 'Non': 'no', 'No': 'no',
+        '是的': 'yes', '是': 'yes', '否': 'no', '不': 'no', 'はい': 'yes', '예': 'yes', '아니오': 'no', 'បាទ': 'yes',
+        'ไม่ใช่': 'no', 'ใช่': 'yes', 'না': 'no', 'हां': 'yes', 'नहीं': 'no', 'نعم': 'yes', 'لا': 'no',
+        'خیر': 'yes', 'بەڵێ': 'yes', 'بله': 'yes', 'أوي': 'yes', 'Нет': 'no', 'НЕТ': 'no', 'Да': 'yes', 'ДA': 'yes',
+        'Ναι': 'yes', 'Hayır': 'no'
+    }
```

Comment:
If I'm following this, are we expecting the values in Touchpoints to be denormalized? It feels like we should be sending normalized values so that we wouldn't need to do this.

Reply:
This is an open question with the Touchpoint developers. This behavior was unexpected by them. We're leaning towards these values being injected by a local translation of the website. This dictionary wouldn't exist if it were only saving the english yes/no values.

Reply:
I'd wonder if that has more to do with how we're overriding the "value" of the options, and that's what is being sent to Touchpoints.

https://github.com/18F/identity-site/blob/f3ce58f6b3902955fcd6816501412d40f8528e67/assets/js/touchpoints_translations.js#L6-L7

I think it'd be better if we'd always send a value of "Yes" or "No" or at least some consistent pair, and only modify the translated text shown to the user. I'm not immediately clear how this would be done, but I also don't know that the Touchpoints team would be able to help us here, since this translation is entirely custom to our project.